src/fidesops/ops/service/email/email_dispatch_service.py

```diff
@@ -15,13 +14,34 @@
     EmailServiceDetails,
     EmailServiceSecrets,
     EmailServiceType,
+    FidesopsEmail,
     SubjectIdentityVerificationBodyParams,
 )
+from fidesops.ops.tasks import DatabaseTask, celery_app
 from fidesops.ops.util.logger import Pii
 
 logger = logging.getLogger(__name__)
 
 
+@celery_app.task(base=DatabaseTask, bind=True)
+def dispatch_email_task(
```

Comment:
After https://github.com/ethyca/fidesops/pull/1274 is merged into this branch, will EMAIL_ERASURE_REQUEST_FULFILLMENT emails also be run in the separate email queue?

Reply:
ok cool, thanks for merging #1274 that should make it easier to coerce for emails of EMAIL_ERASURE_REQUEST_FULFILLMENT type. Is this something you were planning on including here, or in a follow-up.  Also @eastandwestwind's work has been merged with two more email types.